backend/db/seeds.rb

```diff
@@ -28,9 +28,9 @@
 
 10.times do |i|
   Post.create(
-      sender: User.second,
-      receivers: User.last(3),
-      message: Faker::Company.bs,
-      kudos: rand(1..500)
+    sender: User.second,
+    receivers: User.last(3),
```

Comment:
Always the last 3 users?

Reply:
Make it random?

Reply:
I think that would be better, this would also include the sender though.

Reply:
Maybe a bit hacky, suggestions?

```suggestion
    receivers: User.limit(rand(1..4)).order("RANDOM()").where.not(id: 1),
```

Reply:
Looks good to me, you could consider making the values named constants, so you don't have magic numbers in the tests.